### DISCUSSION TITLE
협업 시, mysql 설정 관련 질문 입니다!

### DISCUSSION THREAD

**Discussion by Yonge2** [wanted-7h/hotplace#24] [Q&A]
협업 시, 직접 mysql에 접속하여 세팅을 해주어야 하는 것인가요? 
예를 들면, 계정 생성이나 데이터베이스 생성같은 작업은 협업 시에 보통 어떻게 해결하시는지 궁금합니다.

저번에는 메모리를 사용하는 방식으로 sqlite를 사용해서 편했던 것에 비해 mysql은 조금 달라보여서요ㅜ

**Comment by scarf005** [comment]
> ## Initializing a fresh instance
>
> When a container is started for the first time, a new database with the specified name will be created and initialized with the provided configuration variables. Furthermore, it will execute files with extensions .sh, .sql and .sql.gz that are found in /docker-entrypoint-initdb.d. Files will be executed in alphabetical order. You can easily populate your mysql services by [mounting a SQL dump into that directory](https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-file-as-a-data-volume) and provide [custom images](https://docs.docker.com/reference/builder/) with contributed data. SQL files will be imported by default to the database specified by the MYSQL_DATABASE variable

도커에서 [mysql](https://hub.docker.com/_/mysql)을 실행하고 초기화 스크립트로 환경을 통일시키는 방법이 있을 것 같아요
